CHANGES.rst

```diff
@@ -25,6 +25,11 @@ astropy.extern
 astropy.io.ascii
 ^^^^^^^^^^^^^^^^
 
+astropy.io.asdf
```

Comment:
Note, this is now wrong! SHould be `astropy.io.misc.asdf`!

Reply:
Good catch!